configure.ac

```diff
@@ -457,7 +457,9 @@ AC_C_FLAG([-funwind-tables])
 AC_C_FLAG([-Wall])
 AC_C_FLAG([-Wextra])
 AC_C_FLAG([-Wformat-nonliteral])
-AC_C_FLAG([-Wformat-security])
```

Comment:
didn't the doc say something about clang still needing `-Wformat-security` with `-Wformat=2` or so?